app/views/static_pages/home.html.erb

```diff
@@ -3,5 +3,23 @@
 <h1>Sample App</h1>
 <p>
   This is the home page for the
-  <%= link_to "Ruby on Rails Tutorial", "http://www.railstutorial.org/" %>sample application.
+  <%= link_to "Ruby on Rails Tutorial ", "http://www.railstutorial.org/" %>sample application.
 </p>
+
+<% if logged_in? %>
+  <div class="row">
+    <aside class="col-md-4">
+      <section class="user_info">
+        <%= render "shared/user_info" %>
+      </section>
+      <section class="micropost_form">
+        <%= render "shared/micropost_form" %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3>Micropost Feed</h3>
+      <%= render "shared/feed" %>
+    </div>
+  </div>
+<% else %>
```

Comment:
You seem missing some code handler at here
please check

Reply:
I there is no more code there, I have rechecked with the reference slides, videos and compare with another course mate too.

Reply:
![Screenshot from 2021-09-09 15-19-35](https://user-images.githubusercontent.com/87747744/132641016-03e1af28-711b-4635-88cd-4e8251a829eb.png)

Reply:
I have double checked that no code were added after this point either and my sample app has been working perfectly fine.